src/js/search-bar/search-bar.view.js

```diff
@@ -25,22 +24,35 @@ export class SearchBarView extends AbstractView {
 
     afterViewInit() {
         super.afterViewInit();
-        this.searchBarOverlay = selectDomElement('#search-bar-overlay');
+        this.searchBar = selectDomElement('#search-bar');
         this.searchBtn = selectDomElement('#search-btn');
 
-        document.onkeyup = (event) => {
-            if (event.key === 'Enter') {
+        selectDomElement('#search-input').onkeyup = (event) => {
```

Comment:
Assign this DOM element to a variable like on 27th row. Incapsulate listener binding in separate method if it has some logic